kie-server-parent/kie-server-services/kie-server-services-prometheus/src/main/java/org/kie/server/services/prometheus/PrometheusCustomMetricsSupport.java

```diff
@@ -0,0 +1,130 @@
+package org.kie.server.services.prometheus;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.ServiceLoader;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.jbpm.executor.AsynchronousJobListener;
+import org.jbpm.services.api.DeploymentEventListener;
+import org.kie.api.event.rule.AgendaEventListener;
+import org.kie.dmn.api.core.event.DMNRuntimeEventListener;
+import org.kie.server.services.api.KieContainerInstance;
+import org.optaplanner.core.impl.phase.event.PhaseLifecycleListener;
+
+class PrometheusCustomMetricsSupport {
+
+    private final Map<Class, List<?>> customMetricsInstances;
+
+    private ServiceLoader<PrometheusMetricsProvider> loader;
+    private PrometheusKieServerExtension extension;
```

Comment:
This variable is not needed as it is used to access just static methods.

Reply:
+1 good catch